modules/ROOT/pages/clauses/match.adoc

```diff
@@ -1,80 +1,56 @@
 :description: The `MATCH` clause is used to search for the pattern described in it.
 include::https://raw.githubusercontent.com/neo4j-graphacademy/courses/main/asciidoc/ads/data-analysis.adoc[]
 
-[[query-match]]
 = MATCH
 
-[[match-introduction]]
-== Introduction
-
-The `MATCH` clause allows you to specify the patterns Neo4j will search for in the database.
-This is the primary way of getting data into the current set of bindings.
-For more information about how `MATCH` is used to find patterns (including xref:patterns/variable-length-patterns.adoc#quantified-path-patterns[quantified path patterns], xref:patterns/variable-length-patterns.adoc#quantified-relationships[quantified relationships], and xref:patterns/shortest-paths.adoc[shortest paths]), see the section on xref::patterns/index.adoc[Patterns].
-
-`MATCH` is often coupled to a `WHERE` part which adds restrictions, or predicates, to the `MATCH` patterns, making them more specific.
-The predicates are part of the pattern description, and should not be considered a filter applied only after the matching is done.
-_This means that `WHERE` should always be put together with the `MATCH` clause it belongs to._
-
-`MATCH` can occur at the beginning of the query or later, possibly after a `WITH`.
-If it is the first clause, nothing will have been bound yet, and Neo4j will design a search to find the results matching the clause and any associated predicates specified in any `WHERE` part.
-This could involve a scan of the database, a search for nodes having a certain label, or a search of an index to find starting points for the pattern matching.
-Nodes and relationships found by this search are available as _bound pattern elements,_ and can be used for pattern matching of paths.
-They can also be used in any further `MATCH` clauses, where Neo4j will use the known elements, and from there find further unknown elements.
-
-Cypher is declarative, and so usually the query itself does not specify the algorithm to use to perform the search.
-Neo4j will automatically work out the best approach to finding start nodes and matching patterns.
-Predicates in `WHERE` parts can be evaluated before pattern matching, during pattern matching, or after finding matches.
-However, there are cases where you can influence the decisions taken by the query compiler.
-Read more about indexes in xref:indexes/search-performance-indexes/managing-indexes.adoc[], and more about specifying hints to force Neo4j to solve a query in a specific way in xref::indexes/search-performance-indexes/index-hints.adoc[Planner hints and the USING keyword].
-
+The `MATCH` clause enables you to define specific patterns that the database will search for within its graph structure.
```

Comment:
general note about "will" - i think there's a style guide rule to avoid it where possible. there are many occurrences in this page, but i can also see them being warranted because a query happens at a certain point in time and then the database operations described happen later. i guess it comes down to personal preference :)

Reply:
I will keep it for now :)